### ISSUE TITLE
Overriding Ctrl+C no longer works in Ink 3

### ISSUE THREAD

**Issue by loune** [vadimdemedes/ink#349]
I have an app that requires custom handling of `Ctrl+C` input. The following code works fine in Ink `2.7.1` but doesn't work in `3.0.1`.

```jsx
import React, { useState } from 'react';
import { render, Text, useInput } from 'ink';

const Counter = () => {
  const [ctrlcPresses, setCtrlcPresses] = useState(0);

  useInput((input, key) => {
    if (input === 'c' && key.ctrl) {
      setCtrlcPresses(ctrlcPresses + 1);
    } else {
      process.exit();
    }
  });

  return <Text>Ctrl+C presses: {ctrlcPresses}. Press any other key to exit.</Text>;
};

render(<Counter />, { exitOnCtrlC: false });
```

Any ideas? Thanks

**Comment by vadimdemedes** [comment]
Thanks for reporting, this is indeed a bug. Submitted https://github.com/vadimdemedes/ink/pull/353 to fix it.

**Comment by vadimdemedes** [comment]
Fix released in [3.0.3](https://github.com/vadimdemedes/ink/releases/tag/v3.0.3).
